testfiles/negative/factory-reset-status_unexpected_reset_type

```diff
@@ -0,0 +1 @@
+1:-
```

Comment:
so no json anymore? i remembering, we want an own json file first and on the long run a sub json struction in an initramfs status json.

Reply:
as discussed in the today meeting. for this PR a normal txt file is fine